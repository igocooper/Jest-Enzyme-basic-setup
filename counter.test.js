import React from 'react'
import Counter from './counter';
import { shallow } from 'enzyme';

describe('Count component', () => {
  it('Start with a count of 0', () => {
    const wrapper = shallow(<Counter/>);
    const countState = wrapper.state.count;
    expect(countState).toEqual(0);
  });
});

