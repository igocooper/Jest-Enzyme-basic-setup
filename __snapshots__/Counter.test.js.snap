// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Count component Matches the snapshot 1`] = `
<div>
  <p>
    Current count: 
    0
  </p>
  <button
    className="increment"
    onClick={[Function]}
  >
    +++
  </button>
  <button
    className="decrement"
    onClick={[Function]}
  >
    ---
  </button>
</div>
`;
